package.json

```diff
@@ -59,12 +59,14 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "verify": "yarn prettier-check && yarn eclint && yarn test:ci && yarn stylelint",
+    "verify": "yarn prettier-check && yarn eclint && yarn coverage:ci && yarn stylelint",
     "build:production": "env-cmd -f .env.production yarn build",
     "build:qa": "env-cmd -f .env.qa yarn build",
     "build:int": "env-cmd -f .env.int yarn build",
     "build:development": "env-cmd -f .env.development yarn build",
     "build:demo": "env-cmd -f .env.demo yarn build",
+    "coverage:local": "react-scripts test --coverage --watchAll --env=jest-environment-jsdom-sixteen --testTimeout=120000",
+    "coverage:ci": "cross-env CI=true react-scripts test --coverage --env=jest-environment-jsdom-sixteen --testTimeout=120000",
```

Comment:
Creo que no es necesario tener 2

Reply:
Ah, perdón, si. 

En nuestro caso no es necesario porque la variable de entorno la seteaos en Docker.

Reply:
The local command keeps watching, I think it`s useful when coding. 🤔 Am I missing something?

Reply:
If you are talking about renaming these into tests, it may be better to avoid misunderstandings 😅

Reply:
Right, like this: https://github.com/FromDoppler/doppler-webapp/pull/1593/files#r699546612